<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2021-12-23T22:43:49 IST" name="All Test Suite" finished-at="2021-12-23T22:43:57 IST" duration-ms="7881">
    <groups>
    </groups>
    <test started-at="2021-12-23T22:43:49 IST" name="C:/Users/TG/IdeaProjects/TestN12Framework" finished-at="2021-12-23T22:43:57 IST" duration-ms="7881">
      <class name="tests.validation.WeatherValidationTest">
        <test-method signature="validationCheck()[pri:0, instance:tests.validation.WeatherValidationTest@98637a2]" started-at="2021-12-23T22:43:49 IST" name="validationCheck" finished-at="2021-12-23T22:43:57 IST" duration-ms="7690" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Arrays differ at element [1]: אשדוד !=  expected [אשדוד] but found []]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Arrays differ at element [1]: אשדוד !=  expected [אשדוד] but found []
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:1244)
at org.testng.Assert.assertEquals(Assert.java:1302)
at pages.basedatevalidate.BaseWeatherPageValidate.cityValidation(BaseWeatherPageValidate.java:66)
at tests.validation.WeatherValidationTest.validationCheck(WeatherValidationTest.java:14)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:383)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:344)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:125)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:417)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validationCheck -->
        <test-method is-config="true" signature="clearCookiesAndLocalStorage()[pri:0, instance:tests.validation.WeatherValidationTest@98637a2]" started-at="2021-12-23T22:43:57 IST" name="clearCookiesAndLocalStorage" finished-at="2021-12-23T22:43:57 IST" duration-ms="44" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- clearCookiesAndLocalStorage -->
        <test-method is-config="true" signature="close()[pri:0, instance:tests.validation.WeatherValidationTest@98637a2]" started-at="2021-12-23T22:43:57 IST" name="close" finished-at="2021-12-23T22:43:57 IST" duration-ms="4" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- close -->
      </class> <!-- tests.validation.WeatherValidationTest -->
    </test> <!-- C:/Users/TG/IdeaProjects/TestN12Framework -->
  </suite> <!-- All Test Suite -->
</testng-results>
